charts/k8tz/templates/namespace.yaml

```diff
@@ -1,7 +1,9 @@
+{{- if .Values.namespace -}}
```

Comment:
k8tz must create its own namespace with the special `k8tz.io/controller-namespace` label.

you can read more in my explanation here: https://github.com/k8tz/k8tz/issues/5#issuecomment-950362169